server.js

```diff
@@ -538,14 +538,21 @@ app.get('/question', function(req, res) {
             return res.status(400).send('Question is not available');
         }
 
-        var answeredList = common.getIdsListFromJSONList(questionFound.correctAttempts);
+        var answeredList = common.getIdsListFromJSONList2(questionFound.correctAttempts);
```

Comment:
Whats different about this new function? why did you have to create another function?

Reply:
good question @mehtadarshan1. another thing is if we were to need another one, it should have a better name than `getIdsListFromJSONList2`.

Reply:
The new function literally parses the same json but instead of getting the JSON at "_id" it gets the JSON at "id", essentially just missing an underscore. This will be fixed on @ahmedqarmout2 does the whole ID migration from numbers to that new alphanumeric string that we are using now (which pops up in some snackbars). @ahmedqarmout2 can comment more, should i change this to something else (if so what?) or should I leave this for now and have this merged.

As for the name, if this is temporary than i'd rather just leave the name so it is easier to find and remove later on. But I'm open to suggestions if anyone has any.

Reply:
We need to change the backend to use uuids for students, until then we need to use this second function to handle usernames. The reason the first function does not work is that its expecting uuids not username/display names